Docs/Components_html/lists.html

```diff
@@ -0,0 +1,94 @@
+                <i class="fas fa-bars"></i>
+            </button>
+            <div class="title font-weight-4">nakshatra <span class="sub-title">UI</span></div>
+            <div class="nav-items-box flex-row align-center justify-evenly">
+                <a href="" class="nav-items">Home</a>
+                <a href="" class="nav-items">Docs</a>
+            </div>
+        </nav>
+
+        <aside id="aside-drawer" class="sidebar flex-column position-fixed font-weight-5">
+            <div class="aside-nav-items display-none">
+                <button id="drawer-close-icon" class="close-drawer-btn btn-border-none bg-transparent">
+                    <i class="fas fa-times"></i>
+                </button>
+                <a class="sidebar-lable nav-items" href="">Home</a>
+                <a class="sidebar-lable nav-items" href="">Docs</a>
+            </div>
+            <div class="sidebar-lable font-weight-6">Getting Started</div>
+            <div class="sidebar-items">
+                <a href="">Usage</a>
+            </div>
+            <div class="sidebar-items">
+                <a href="./typography.html">Typography</a>
+            </div>
+
+            <div class="sidebar-lable font-weight-6">Components</div>
+            <ul>
+                <li class="sidebar-items">
+                    <a href="./alert.html">Alert</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="">Avatar</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="">Badge</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./buttons.html">Button</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="">Card</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./image.html">Image</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./inputs.html">Input</a>
+                </li>
+                <li class="sidebar-items">
+                    <a href="./lists.html">List</a>
+                </li>
+            </ul>
+        </aside>
+        <main class="main-section flex-column">
+            <section>
+                <h1 class="headings main-heading font-weight-4"></h1>
+                <p class="heading-about font-weight-3"> 
+                    
+                </p>
+            </section>
+            <section class="component-section flex-column">
+                <div>
+                    <h2 class="heading sub-heading font-weight-4"></h2>
+                    <p class="component-info">
+                        
+                    </p>
+                </div>
+
+                <div class="component-display-box font-weight-5">
+                   
+                </div>
+
+                
+            </section>
```

Comment:
unnecessary spaces can be removed.

Reply:
I don't know what happened here but in here there are components which is not showing here but everywhere else it's showing components even in github code section you will find them.